internal/consensus/peer_state.go

```diff
@@ -19,6 +19,8 @@ import (
 var (
 	ErrPeerStateHeightRegression = errors.New("peer state height regression")
 	ErrPeerStateInvalidStartTime = errors.New("peer state invalid startTime")
+	ErrPeerStateNilVote          = errors.New("peer state nil vote")
+	ErrPeerStateInvalidVoteIndex = errors.New("peer state invalid vote index")
```

Comment:
Could we perhaps have a more descriptive error message i.e. `peer sent a vote with an invalid index`

Reply:
yes, updated!